ansible/group_vars/lab/lab.yml

```diff
@@ -54,3 +54,8 @@ fanout_admin_password: "fanoutadminpassword"
 
 mgmt_admin_user: "mgmtadminuser"
 mgmt_admin_password: "mgmtadminpassword"
+
+# Flag for run UTs with lab TACACS server
+test_with_lab_tacacs_server: false
```

Comment:
>test_with_lab_tacacs_server [](http://example.com/codeflow?start=0&length=27)

If we merge this PR, does it impact KVM test?

Reply:
No, this flag defined in lab.yml which means only UT run on lab device will be impact by this flag.
Every lab will inherit this file and override this flag.

Reply:
Please change UT in the title and description. UT normally means component or function level test in a build system, not in lab test or KVM test.